common/Windows/WinHostSys.cpp

```diff
@@ -47,6 +48,16 @@ long __stdcall SysPageFaultExceptionFilter(EXCEPTION_POINTERS* eps)
 	{
 		return EXCEPTION_CONTINUE_SEARCH;
 	}
+#else
+  try
+  {
+    return DoSysPageFaultExceptionFilter(eps);
+  }
+  catch (...)
```

Comment:
This isn't going to work, unless your compiler transforms SEH exceptions into C++ exceptions. As far as I'm aware, MSVC is the only one which does this, but I could be wrong (although it's not really something you want, and it's disabled by default).

Fortunately, there's other options, like using vectored exception handlers, which we use on 64-bit anyway.

Reply:
afaik gcc handles c++ exception as seh or sjlj